<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connexion</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblUsername">
        <property name="text">
         <string>Nom d'utilisateur : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtUsername"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblPassword">
        <property name="text">
         <string>Mot de passe :       </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtPassword"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnConnection">
        <property name="text">
         <string>Connexion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnInscription">
        <property name="text">
         <string>Inscription</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLeave">
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
